test_files/fake_goog_reflect.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * @fileoverview A simple fake version of goog.reflect, for testing compilation.
+ */
+
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+var goog = goog || {};
```

Comment:
do you need this?

Reply:
Switched to goog.module, which is slightly different from the real version, but operationally much simpler.